azure/Guidelines.md

```diff
@@ -0,0 +1,310 @@
+# Microsoft Azure REST Design Guidelines
+
+## History
+
+<dl>
+<dd>2020-Mar-31 v3.0</dd>
+<dt>Version 3.0 of the Azure REST API Guidelines was copied from Sharepoint
+into Markdown and uploaded to GitHub as the basis for future improvements.</dt>
+</dl>
+
+## Introduction
+
+The Azure REST API guidelines are an extension of the Microsoft-wide [OneAPI guidelines][1] (which historically drew heavily from an earlier version of the Azure REST API guidelines). Readers of this document are assumed to be also reading the [OneAPI guidelines][1] and be familiar with them.  Azure guidance is a superset of OneAPI guidelines and services should follow them *except* where this document outlines specific differences or exceptions to those guidelines. This document does contain additional Azure-specific guidance and additional details.
+
+The OneAPI guidelines are available on Github at [https://github.com/microsoft/api-guidelines][1].
+
+Highlights of the OneAPI guidelines include:
+
+* **REST fundamentals**.  Guidance around REST basics such as HTTP headers, verbs, status codes, and other similar areas.
+* **Versioning**.  The guidance for versioning covers the externally facing mechanics of versioning and compatibility guarantees between versions.  Guidance for when API owners should increment versions is also present.
+* **CORS**.  Guidance around the use of CORS.
+* **Authentication**.  Guidance around the authentication and its use.
+* **Encoding**.  Guidance around the use of JSON encoding and standards.
+* **Error responses**.  A standard format for error responses is closed.
+* **Dates and times**.  Detailed guidance around date and time formats and encoding is provided.
+* **Relationship to OData**.  Guidance around the relationship between OData and REST APIs.
+* **JSON representation**.  JSON is preferred over XML and other formats.
+* **Long running operations**.  The OneAPI guidelines are updated and simplified over the previous Azure guidelines.
+* **Push notifications via Webhooks**.
+
+### Updating from previous versions of the Azure API guidelines
```

Comment:
We should just delete this as its refering to an old version from 4 years ago

Reply:
Ack = removing the whole section.